spec/source/manager_spec.rb

```diff
@@ -25,19 +25,19 @@ module Pod
       end
 
       it 'returns all the sources' do
-        @sources_manager.all.map(&:name).should == %w(master test_cdn_repo_local test_empty_dir_repo test_prefixed_repo test_repo)
+        @sources_manager.all.map(&:name).should == %w(artsy test_cdn_repo_local test_empty_dir_repo test_prefixed_repo test_repo test_repo_without_specs_dir trunk)
       end
 
       it 'includes all sources in an aggregate for a dependency if no source is specified' do
         dependency = Dependency.new('JSONKit', '1.4')
         aggregate = @sources_manager.aggregate_for_dependency(dependency)
-        aggregate.sources.map(&:name).should == %w(master test_cdn_repo_local test_empty_dir_repo test_prefixed_repo test_repo)
+        aggregate.sources.map(&:name).should == %w(artsy test_cdn_repo_local test_empty_dir_repo test_prefixed_repo test_repo test_repo_without_specs_dir trunk)
       end
 
       it 'includes all sources in an aggregate for a dependency if non-existent source is specified' do
         dependency = Dependency.new('JSONKit', '1.4', :source => 'https://url/to/nonexistent/specs.git')
         aggregate = @sources_manager.aggregate_for_dependency(dependency)
-        aggregate.sources.map(&:name).should == %w(master test_cdn_repo_local test_empty_dir_repo test_prefixed_repo test_repo)
+        aggregate.sources.map(&:name).should == %w(artsy test_cdn_repo_local test_empty_dir_repo test_prefixed_repo test_repo test_repo_without_specs_dir trunk)
```

Comment:
is there a reason `trunk` is always last? not that it's an issue, but I think they should be in the order they were declared

Reply:
It might need to be taken care of yes because lint and Molinillo might discover pods in the order the sources were passed in.

Reply:
but this might not be the place to do this ordering. Maybe it gets re-arranged somewhere else.

Reply:
yeah, but something has changed here because all the test cases now have trunk at the end (where master used to be first). not sure where/why that happened

Reply:
I think this is repo dir sort order.

Reply:
ah yea this looks sorted in the test so I think this is fine.

Reply:
I hate myself for suggesting this, but... Maybe `_trunk`?

Reply:
You know what, never mind.  
If we want to enforce some order for the dep resolver, it has to be done in code related to dep resolution and not in the source manager.

Reply:
correct. I wouldnt bother with the order here.